docs/src/main/paradox/reference.md

```diff
@@ -67,14 +67,22 @@ See also GeoMesa [st_envelope](https://www.geomesa.org/documentation/user/spark/
 Convert an extent to a Geometry. The extent likely comes from @ref:[`st_extent`](reference.md#st-extent) or @ref:[`rf_extent`](reference.md#rf-extent).
 
 
-### rf_spatial_index
+### rf_xz2_index
 
-    Long rf_spatial_index(Geometry geom, CRS crs)
-    Long rf_spatial_index(Extent extent, CRS crs)
-    Long rf_spatial_index(ProjectedRasterTile proj_raster, CRS crs)
+    Long rf_xz2_index(Geometry geom, CRS crs)
+    Long rf_xz2_index(Extent extent, CRS crs)
+    Long rf_xz2_index(ProjectedRasterTile proj_raster)
```

Comment:
below we should mention and link to the raster read section about spatial index parititoning.